src/Coalesce.React/common-components/src/js/templateController.js

```diff
@@ -80,7 +80,7 @@ export function loadTemplateByName(name, source, version)
 
 export function registerTemplate(key)
 {
-  return fetch(`${karafRootAddr}/templates/${key}`, {
+  return fetch(`${karafRootAddr}/templates`, {
```

Comment:
This is going to have the same issue as the POST endpoint.

Reply:
Actually I believe this needs to be reverted. Looking at the API change it still requires a key.